packages/dds/matrix/src/permutationvector.ts

```diff
@@ -234,12 +234,12 @@ export class PermutationVector extends Client {
             0x028 /* "'localSeq' for op being resubmitted must be <= the 'localSeq' of the last submitted op." */);
 
         // TODO: In theory, the MergeTree should be able to map the (position, refSeq, localSeq) from
-        //       the original operation to the current position for resubmitting.  This is probably the
+        //       the original operation to the current position for undo/redo scenarios.  This is probably the
```

Comment:
Here's the changed comment, let me know if this is what you expected or if there's something else I should change about it.

Reply:
yeah it seems reasonable enough. You could probably make the statement a bit weaker (I'm not 100% convinced the support isn't already there and we just didn't go through the undo/redo scenario closely enough to realize the correct code adjustment), but good enough for posterity.